File: Programmers/연습문제/Level 1/없는 숫자 더하기/Solution.js
function solution(numbers) {
    const answer = findAnswer(numbers);
    return answer;
}

function sum(numbers) {
    let result = 0;
    for (const number of numbers) {
        result += number;
    }
    return result;
}

function findAnswer(numbers) {
    return 45 - sum(numbers);
}

// TC
const numbers = [1,2,3,4,6,7,8,0]
console.log(solution(numbers)); // 14

File: Programmers/연습문제/Level 1/없는 숫자 더하기/Solution.swift
import Foundation

func solution(_ numbers: [Int]) -> Int {
    return (0...9).filter{!numbers.contains($0)}.reduce(0, +)
}

// TC
let numbers = [1, 2, 3, 4, 6, 7, 8, 0]
print(solution(numbers)) // 14
